python/pyarrow/tests/test_exec_plan.py

```diff
@@ -209,44 +209,27 @@ def test_filter_table_errors():
     })
 
     with pytest.raises(pa.ArrowTypeError):
-        ep._filter_table(
-            t, pc.divide(pc.field("a"), pc.scalar(2)),
-            output_type=pa.Table
```

Comment:
Since we only ever used `output_type=pa.Table` option for this keyword in `_filter_table`, I removed the keyword and simplified the tests